tests-dev/test_cases/exp_conf/2020_CAPE.IN

```diff
@@ -0,0 +1,176 @@
+rm -fr INPUT RESTART
+if [ $NPX = 97 ]; then
+  inputdir=FV3_input_data
+elif [ $NPX = 49 ]; then
+  inputdir=FV3_input_data48
+elif [ $NPX = 193 ]; then
+  inputdir=FV3_input_data192
+elif [ $NPX = 385 ]; then
+  inputdir=FV3_input_data384
+elif [ $NPX = 769 ]; then
+  inputdir=FV3_input_data768
+fi
+
+echo "inputdir=$inputdir,NPX=$NPX,NESTED=$NESTED"
+
+OPNREQ_TEST=${OPNREQ_TEST:-false}
+V2_SFC_FILE=${V2_SFC_FILE:-false}
+HSD_CASE=${HSD_CASE:-false}
+SUFFIX=${RT_SUFFIX}
+if [ $WARM_START = .false. ]; then
+  mkdir INPUT RESTART
+  if [ "$NESTED" = .true. ]; then
+    cp -r @[INPUTDATA_ROOT]/${inputdir}/INPUT_L127_nested/* ./INPUT/.
+  elif [ "$V2_SFC_FILE" = "true" ]; then
+    cp -r @[INPUTDATA_ROOT]/${inputdir}/INPUT_L127_v2_sfc/* ./INPUT/.
+  elif [ "$HSD_CASE" = "true" ]; then
+    cp -r @[INPUTDATA_ROOT]/HSD_input_data/${inputdir}/INPUT_L127_v2_sfc/* ./INPUT/.
+  else
+    cp -r @[INPUTDATA_ROOT]/${inputdir}/INPUT_L127_gfsv17/* ./INPUT/.
+  fi
+else
+  mkdir INPUT RESTART
+
+  if [[ ${OPNREQ_TEST} == true ]]; then
+    SUFFIX=${BL_SUFFIX}
+  fi
+
+  # WAVE restart file for ATMW ORT tests
+  if [[ $CPLWAV == .true. ]]; then
+    if [[ $med_model == cmeps ]] && [[ $RUNTYPE == continue ]]; then
+      # CMEPS restart and pointer files
+      RFILE=ufs.atmw.cpl.r.${RESTART_FILE_SUFFIX_SECS}.nc
+      cp  ../${DEP_RUN}${SUFFIX}/RESTART/${RFILE} .
+      ls -1 ${RFILE}>rpointer.cpl
+    fi
+    RFILE=ufs.atmw.ww3.r.${RESTART_FILE_SUFFIX_SECS}
+    cp ../${DEP_RUN}${SUFFIX}/${RFILE} .
+  fi
+
+  if [ $MODEL_INITIALIZATION = "false" ]; then
```

Comment:
I don't see why MODEL_INITIALIZATION is needed. Isn't this just if it is a DEP_RUN?

Reply:
i can look into trimming this down in a follow-up

Reply:
sounds like new PR will come soon to update documentation and container.